packages/html-to-slate-ast/src/index.ts

```diff
@@ -32,13 +38,18 @@ const ELEMENT_TAGS: Record<
   TR: () => ({ type: 'table_row' }),
   TD: () => ({ type: 'table_cell' }),
   TH: () => ({ type: 'table_cell' }),
-  IMG: (el) => ({
-    type: 'link',
-    href: el.getAttribute('src'),
-    title: Boolean(el.getAttribute('alt')) ? el.getAttribute('alt') : '(Image)',
-    openInNewTab: true,
-    children: [{ text: el.getAttribute('src') }],
-  }),
+  IMG: (el) => {
+    const href = el.getAttribute('src');
+    if (href === null) return {};
+    return {
+      type: 'link',
+      href: sanitizeUrl(href),
+      title: Boolean(el.getAttribute('alt'))
+        ? el.getAttribute('alt')
+        : '(Image)',
```

Comment:
Just saw that when users change alt text to images in GDocs, `title` is assigned to `title` and `description` to `alt`
So maybe fallback to `title` if `alt` is not present?
Other than that LGTM 🚀